js/react/util/BackboneWrapper.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+interface IProps {
```

Comment:
Haha, this might be contentious but the TypeScript community has a convention not to use `I` prefixes for interfaces:

- https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names
- https://github.com/Microsoft/TypeScript-Handbook/issues/121

Since we seem to agree to prefer not using classes, this shouldn’t be a big issue for us. Let’s discuss this in person 😄

Reply:
Fixed! I only did it because `tslint` defaulted that way.